system-int-test/test/cmr/system_int_test/search/subscription/subscription_search_test.clj

```diff
@@ -205,6 +205,32 @@
             (search/find-concepts-in-format
              nil :subscription {} {:url-extension "atom"}))))))
 
+(deftest search-for-subscription-type-field
```

Comment:
This is the wrong test. It didn't test the JSON response fields.

Reply:
Adjusted test to verify type field for JSON response. Let me know if it needs to be adjusted further!